Michal

michalborkowski96 opened the pull request:
#37

michalborkowski96 committed:
C struct print

manio143 commented:
Michał zrób 

     git rebase -i HEAD~7

I przy każdym commicie który jest niepotrzebny zaznacz drop zgodnie z instrukcją. Następnie zrób force push, żeby zmiany pojawiły sie w tym PRze.

Najlepiej, żeby nie robić merge z masterem tylko rebase na niego

jbujak committed:
Code review and minor fixes to structs generating

Change use of string to variants in type indication in generator_c_struct_dependence_sort
Change generated type of ptd::rec to ImmT
Append 'field' to field name in structure

jbujak commented:
Dodałem kilka małych poprawek:
* Dla ptd::rec chcemy zostawić taką obsługę, jak była, więc zmieniłem generowany typ na ImmT.
* Zmieniłem te stringi na warianty ‒ wiele to nie daje, ale zabezpiecza chociaż przed przypisaniem stringa, który nic nie oznacza
* Dodałem sufiks field do nazw pól w strukturach

jbujak committed:
Fix test for generated structures

Structure should be generated only for own records
